httpie/config.py

```diff
@@ -84,7 +84,8 @@ class Config(BaseConfigDict):
     about = 'HTTPie configuration file'
 
     DEFAULTS = {
-        'default_options': []
+        'default_options': [],
+        'extra_site_dirs': [],
```

Comment:
We could detect when running in a Homebrew installation and set `extra_site_dirs` here to a useful default such as the example in `README.rst`.  I'm not sure if it's a good idea.